x-pack/plugins/triggers_actions_ui/public/application/components/builtin_alert_types/threshold/visualization.tsx

```diff
@@ -175,12 +175,12 @@ export const ThresholdVisualization: React.FunctionComponent<Props> = ({
     threshold,
     startVisualizationAt,
   ]);
-  /* eslint-enable react-hooks/exhaustive-deps */
 
   if (!charts || !uiSettings || !dataFieldsFormats) {
```

Comment:
```suggestion
  if (!charts || !charts.theme || !uiSettings || !dataFieldsFormats) {
```

Reply:
It looks like this is unnecessary but happy to add it if you'd like.

![image](https://user-images.githubusercontent.com/19007109/85477570-30f49c80-b580-11ea-9c2c-9204b24e008b.png)

Reply:
It looks fine as it is - thanks!